JustSaying.Sample.Restaurant.OrderingApi/JustSaying.Sample.Restaurant.OrderingApi.csproj

```diff
@@ -20,4 +23,10 @@
     <ProjectReference Include="..\JustSaying\JustSaying.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Update="appsettings.json">
```

Comment:
Could you simplify this to this form?

```xml
<Content Update="appsettings.json" CopyToOutputDirectory="PreserveNewest" />
```

Reply:
Done - didnt know the elements could be converted to properties like that